content/en/blog/_posts/2025-04-23-userns-enabled-by-default/index.md

```diff
@@ -0,0 +1,282 @@
+---
+layout: blog
+title: "Kubernetes 1.33: User Namespaces enabled by default!"
+date: 2025-04-23
+draft: true
+slug: userns-enabled-by-default
+author: >
+  Rodrigo Campos Catelin (Microsoft),
+  Giuseppe Scrivano (Red Hat),
+  Sascha Grunert (Red Hat)
+---
+
+In Kubernetes 1.33 support for user namespaces is enabled by default. This means
+that, when the stack requirements are met, pods can opt-in to use user
+namespaces. There is no need to enable any Kubernetes feature flag anymore!
```

Comment:
nit: I guess what this says is `To use the feature there is no need to enable any Kubernetes feature flag anymore!`. Maybe this is obvious, but actually I was surprised by the statement. OTOH, it is longer, so as you prefer.

Reply:
Fixed, thanks!